### PR TITLE
Feat/va components fix

### PR DESCRIPTION
**Pull request by AlexanderBoriskin**
Related to #679 (vuestic-admin issue):
1) Bug 1 - added property 'is-collapse' to provide collapse behaviour for va-button
2) Bug 2 - added property 'delay' and updated popover demo to provide possibility to set delay from user side
3) Bug 3 - added property 'default-header' to remove va-table styles from header; 'row-class' property provides necessary styles for rows; further (after merging PR data table presentation should be updated
4) Bug 5 - area of prepend slot for input now placed inside input container
5) Bug 9 - va-input styles applied to custom input inside chrome picker (see va-advanced-color-picker component)

### CODE REVIEW

File: src/components/vuestic-components/va-button/VaButton.vue [hunk 1]
```diff
@@ -265,11 +269,12 @@ export default {
   &--default {
     color: $white;
 
-    &:hover {
+    &.va-button--hover {
```
Comment (Kreezag) [inline src/components/vuestic-components/va-button/VaButton.vue]: On styleguide should be `&--hover`

File: src/components/vuestic-components/va-data-table/VaDataTable.demo.vue [hunk 2]
```diff
@@ -7,20 +7,28 @@
...
+    VaDataTableCustomStyling,
     VaDataTableStaticDataExample,
     VaDataTableApiModeExample,
   },
 }
 </script>
 
 <style lang="scss">
+  @import '../../vuestic-sass/resources/resources';
+  @import './../../vuestic-sass/global/typography';
```
Comment (Kreezag) [inline src/components/vuestic-components/va-data-table/VaDataTable.demo.vue]: Please, leave a comment about these imports, if these imports really need it.

File: src/components/vuestic-components/va-collapse/VaCollapse.demo.vue [hunk 3]
```diff
@@ -10,7 +10,11 @@
     </VbCard>
     <VbCard title="Custom header">
       <va-collapse style="width: 400px;" customHeader>
-        <va-button slot="header" style="width: 100%">
+        <va-button
+          slot="header"
+          style="width: 100%"
+          type="collapse"
```
Comment (Kreezag) [inline src/components/vuestic-components/va-collapse/VaCollapse.demo.vue]: Wontfix:
Please, change prop type to `is-collapse` boolean prop.  In this case, type look worse

**Comment by asvae** [comment]
Merged manually

**Pull request closed without merging**